kube-rbac-proxy/publish.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
```

Comment:
I feel publishing the image should be doable only in ci/cd, and then the way to do it is like https://github.com/lumigo-io/lumigo-kubernetes-operator/blob/db46e0aed46cd32c69437761a0104b18264d1ccc/.github/workflows/build-test-release.yml#L426